src/python/bot/fuzzers/dictionary_manager.py

```diff
@@ -129,6 +133,59 @@ def merge_dictionary_files(original_dictionary_path,
   utils.write_data_to_file(merged_dictionary_data, merged_dictionary_path)
 
 
+def correct_if_needed(dict_path):
+  """Corrects obvious errors such as missing quotes in a dictionary."""
+
+  def fix_line(line):
+    """Correct a single dictionary line."""
+    # Ignore blank and comment lines.
+    if not line or line.startswith('#'):
```

Comment:
how about whitespace chars that come before "#" ? should we call strip() on the line first?